api/src/pdc/services/auth/dex/protofiles/api.proto

```
@@ -0,0 +1,284 @@
+syntax = "proto3";
+
+package api;
```

Buf (1.47.2)

Files with package "api" must be within a directory "api" relative to root but were in directory "api/src/pdc/services/auth/dex/protofiles".

(PACKAGE_DIRECTORY_MATCH)